egs/discophone/v1/run.sh

```diff
@@ -0,0 +1,377 @@
+#!/bin/bash
```

Comment:
I don't understand why this script is rendered as fully new - I'm pretty there is a `run.sh` already... can you tell me what are the changes? I noticed you removed the `&` and `wait` commands - are you running it on a single machine? Do you have access to a grid (with sge/qsub, slurm, etc.)? It would run much faster that way.

Reply:
> I don't understand why this script is rendered as fully new - I'm pretty there is a `run.sh` already... can you tell me what are the changes? I noticed you removed the `&` and `wait` commands - are you running it on a single machine? Do you have access to a grid (with sge/qsub, slurm, etc.)? It would run much faster that way.

I have access to slurm server. I commented & and wait for some reasons but I can't simply describe why. It didn't matter to me, because when I run experiments, I run one job for one babel/gp language and multiple jobs at the same time. So it is approximately equal to parallel running.

I updated run.sh to the one same to your version.